### PR TITLE
[do-not-merge] apply movefmt to framework code

### PR DESCRIPTION
**Pull request by rahxephon89**
## Description

This PR shows the result after applying the movefmt to the framework code. Please feel free to leave feedback!

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 1]
```diff
@@ -175,14 +181,17 @@ module aptos_framework::account {
...
 
     /// Only called during genesis to initialize system resources for this module.
     public(friend) fun initialize(aptos_framework: &signer) {
         system_addresses::assert_aptos_framework(aptos_framework);
-        move_to(aptos_framework, OriginatingAddress {
-            address_map: table::new(),
-        });
+        move_to(
+            aptos_framework,
+            OriginatingAddress {address_map: table::new(),}
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:193]: This looks strange to me.

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 2]
```diff
@@ -196,11 +205,15 @@ module aptos_framework::account {
...
+        assert!(
+            !exists<Account>(new_address),
+            error::already_exists(EACCOUNT_ALREADY_EXISTS)
+        );
 
         // NOTE: @core_resources gets created via a `create_account` call, so we do not include it below.
         assert!(
-            new_address != @vm_reserved && new_address != @aptos_framework && new_address != @aptos_token,
+            new_address != @vm_reserved && new_address != @aptos_framework && new_address
+                != @aptos_token,
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:216]: This line break does not seem correct.

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 3]
```diff
@@ -333,30 +357,45 @@ module aptos_framework::account {
...
+            let from_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(
+                &from_pk
+            );
             assert!(
                 account_resource.authentication_key == from_auth_key,
                 error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY)
             );
-        } else {
+        }
+        else {
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:391]: The position of this `else` is off

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 4]
```diff
@@ -745,27 +892,23 @@ module aptos_framework::account {
...
-                addr == @0x3 ||
-                addr == @0x4 ||
-                addr == @0x5 ||
-                addr == @0x6 ||
-                addr == @0x7 ||
-                addr == @0x8 ||
-                addr == @0x9 ||
-                addr == @0xa,
-            error::permission_denied(ENO_VALID_FRAMEWORK_RESERVED_ADDRESS),
+            addr == @0x1 || addr == @0x2 || addr == @0x3 || addr == @0x4 || addr == @0x5
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:903]: not sure we could do better here?

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 2 above)
Comment (JohnChangUK) [reply aptos-move/framework/aptos-framework/sources/account.move:216]: I prefer
```
new_address != @vm_reserved 
    && new_address != @aptos_framework 
    && new_address != @aptos_token,
```

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 5]
```diff
@@ -275,9 +290,15 @@ module aptos_framework::account {
...
-    public(friend) fun rotate_authentication_key_internal(account: &signer, new_auth_key: vector<u8>) acquires Account {
+    public(friend) fun rotate_authentication_key_internal(
+        account: &signer,
+        new_auth_key: vector<u8>
+    ) acquires Account {
         let addr = signer::address_of(account);
-        assert!(exists_at(addr), error::not_found(EACCOUNT_DOES_NOT_EXIST));
+        assert!(
+            exists_at(addr),
+            error::not_found(EACCOUNT_DOES_NOT_EXIST)
```
Comment (JohnChangUK) [inline aptos-move/framework/aptos-framework/sources/account.move:300]: Think we should have a comma at the end of every break line. i.e.
```
assert!(
    exists_at(addr),
    error::not_found(EACCOUNT_DOES_NOT_EXIST),
);
```

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move [hunk 6]
```diff
@@ -35,10 +34,16 @@ module aptos_framework::aggregator {
     }
 
     /// Adds `value` to aggregator. Aborts on overflowing the limit.
-    public native fun add(aggregator: &mut Aggregator, value: u128);
+    public native fun add(
```
Comment (JohnChangUK) [inline aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:37]: Is this necessary? Feel like files will end up being double the code lines due to this.

File: aptos-move/framework/aptos-framework/sources/aptos_coin.move [hunk 7]
```diff
@@ -153,14 +182,22 @@ module aptos_framework::aptos_coin {
...
     }
 
     // This is particularly useful if the aggregator_factory is already initialized via another call path.
     #[test_only]
-    public fun initialize_for_test_without_aggregator_factory(aptos_framework: &signer): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
+    public fun initialize_for_test_without_aggregator_factory(aptos_framework: &signer)
+        : (
+        BurnCapability<AptosCoin>,
+        MintCapability<AptosCoin>
+    ) {
```
Comment (JohnChangUK) [inline aptos-move/framework/aptos-framework/sources/aptos_coin.move:200]: Seems off, should be something like

```
public fun initialize_for_test_without_aggregator_factory(
    aptos_framework: &signer,
): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
```

File: aptos-move/framework/aptos-framework/sources/account.spec.move [hunk 8]
```diff
@@ -590,49 +725,42 @@ spec aptos_framework::account {
...
+        ensures result_2 == SignerCapability {account: resource_addr};
     }
 
     /// Check if the bytes of the new address is 32.
     /// The Account does not exist under the new address before creating the account.
     /// The system reserved addresses is @0x1 / @0x2 / @0x3 / @0x4 / @0x5  / @0x6 / @0x7 / @0x8 / @0x9 / @0xa.
     spec create_framework_reserved_account(addr: address): (signer, SignerCapability) {
         aborts_if spec_is_framework_address(addr);
-        include CreateAccountAbortsIf {addr};
+        include CreateAccountAbortsIf { addr };
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.spec.move:736]: Why does the tool add spaces here, which seems to be inconsistent with the case above where the spaces are removed?

File: aptos-move/framework/aptos-framework/sources/aptos_account.spec.move [hunk 9]
```diff
@@ -121,105 +131,133 @@ spec aptos_framework::aptos_account {
...
-                !account::exists_at(recipients[i]) && (recipients[i] == @vm_reserved || recipients[i] == @aptos_framework || recipients[i] == @aptos_token);
-        ensures forall i in 0..len(recipients):
-                (!account::exists_at(recipients[i]) ==> !length_judgment(recipients[i])) &&
-                    (!account::exists_at(recipients[i]) ==> (recipients[i] != @vm_reserved && recipients[i] != @aptos_framework && recipients[i] != @aptos_token));
+        aborts_if exists i in 0..len(recipients): !account::exists_at(recipients[i]) && length_judgment(
+            recipients[i]
+        );
+        aborts_if exists i in 0..len(recipients): !account::exists_at(recipients[i]) && (
+            recipients[i] == @vm_reserved || recipients[i] == @aptos_framework || recipients[
+                i
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:155]: This `i` seems to be in a separate line?

File: aptos-move/framework/move-stdlib/tests/vector_tests.move [hunk 10]
```diff
@@ -84,7 +104,7 @@ module std::vector_tests {
         assert!(!V::is_empty(&v1), 0);
         assert!(V::length(&v1) == 8, 1);
         let i = 0;
-        while (i < 8) {
+        while (i <8) {
```
Comment (rahxephon89) [inline aptos-move/framework/move-stdlib/tests/vector_tests.move:107]: This does not look correct

File: aptos-move/framework/move-stdlib/sources/fixed_point32.move [hunk 11]
```diff
@@ -268,26 +282,26 @@ module std::fixed_point32 {
...
-        } else {
-            (val.value - fractional + one) >> 32
+        if (fractional < boundary) {(val.value - fractional) >> 32} else {(val.value - fractional + one) >> 32
         }
     }
 
     // **************** SPECIFICATIONS ****************
 
-    spec module {} // switch documentation context to module level
+    spec module {
```
Comment (rahxephon89) [inline aptos-move/framework/move-stdlib/sources/fixed_point32.move:303]: This looks strange to me.

File: aptos-move/framework/move-stdlib/sources/configs/features.spec.move [hunk 12]
```diff
@@ -40,11 +52,24 @@ spec std::features {
     }
 
     spec fun spec_contains(features: vector<u8>, feature: u64): bool {
-        ((int2bv((((1 as u8) << ((feature % (8 as u64)) as u64)) as u8)) as u8) & features[feature/8] as u8) > (0 as u8)
-            && (feature / 8) < len(features)
-    }
-
-    spec change_feature_flags_internal(framework: &signer, enable: vector<u64>, disable: vector<u64>) {
+        (
```
Comment (rahxephon89) [inline aptos-move/framework/move-stdlib/sources/configs/features.spec.move:55]: This looks strange to me.

File: aptos-move/framework/move-stdlib/sources/bit_vector.move [hunk 13]
```diff
@@ -145,95 +173,128 @@ module std::bit_vector {
...
-                invariant forall j in start_index..index: j < vector::length(bitvector.bit_field);
-            };
-            index < bitvector.length
-        }) {
+        while (
+            {
+                spec {
+                    invariant index >= start_index;
+                    invariant index == start_index || is_index_set(bitvector, index -
+                            1);
```
Comment (rahxephon89) [inline aptos-move/framework/move-stdlib/sources/bit_vector.move:192]: This line break here looks strange to me.

File: aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move [hunk 14]
```diff
@@ -4,23 +4,11 @@ spec aptos_std::table_with_length {
...
-            map_del_must_exist = remove,
-            map_borrow = borrow,
-            map_borrow_mut = borrow_mut,
-            map_borrow_mut_with_default = borrow_mut_with_default,
-            map_spec_get = spec_get,
-            map_spec_set = spec_set,
-            map_spec_del = spec_remove,
-            map_spec_len = spec_len,
-            map_spec_has_key = spec_contains;
+        pragma intrinsic = map, map_new = new, map_destroy_empty = destroy_empty, map_len =
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move:7]: We should preserve the original line break here.

File: aptos-move/framework/aptos-framework/sources/delegation_pool.move [hunk 15]
```diff
@@ -6,7 +7,7 @@ proportionally to their stake and provided the same stake-management API as the
 The main accounting logic in the delegation pool contract handles the following:
 1. Tracks how much stake each delegator owns, privately deposited as well as earned.
 Accounting individual delegator stakes is achieved through the shares-based pool defined at
-<code>aptos_std::pool_u64</code>, hence delegators own shares rather than absolute stakes into the delegation pool.
+<code>aptos_std::pool_u64<ode>, hence delegators own shares rather than absolute stakes into the delegation pool.
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/delegation_pool.move:10]: This is a bug

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move [hunk 16]
```diff
@@ -24,8 +24,7 @@ module aptos_framework::aggregator {
     /// Represents an integer which supports parallel additions and subtractions
     /// across multiple transactions. See the module description for more details.
     struct Aggregator has store {
-        handle: address,
-        key: address,
+        handle: address, key: address,
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:27]: This is very weird

Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:27]: It wraps variables, but does nothing with comments?

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 17]
```diff
@@ -231,8 +246,8 @@ module aptos_framework::account {
                 guid_creation_num,
                 coin_register_events,
                 key_rotation_events,
-                rotation_capability_offer: CapabilityOffer { for: option::none() },
-                signer_capability_offer: CapabilityOffer { for: option::none() },
+                rotation_capability_offer: CapabilityOffer {for: option::none()},
+                signer_capability_offer: CapabilityOffer {for: option::none()},
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/account.move:250]: No spaces?

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 18]
```diff
@@ -333,30 +357,45 @@ module aptos_framework::account {
...
+            );
             assert!(
                 account_resource.authentication_key == from_auth_key,
                 error::unauthenticated(EWRONG_CURRENT_PUBLIC_KEY)
             );
-        } else if (from_scheme == MULTI_ED25519_SCHEME) {
-            let from_pk = multi_ed25519::new_unvalidated_public_key_from_bytes(from_public_key_bytes);
-            let from_auth_key = multi_ed25519::unvalidated_public_key_to_authentication_key(&from_pk);
+        }
+        else if (from_scheme == MULTI_ED25519_SCHEME) {
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/account.move:379]: Why is this two separate lines?

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 19]
```diff
@@ -462,64 +514,104 @@ module aptos_framework::account {
...
-            option::is_some(&account_resource.rotation_capability_offer.for),
+            option::is_some(
+                &account_resource.rotation_capability_offer.for
+            ),
             error::not_found(ENO_SIGNER_CAPABILITY_OFFERED),
         );
-        *option::borrow(&account_resource.rotation_capability_offer.for)
+        *option::borrow(
+            &account_resource.rotation_capability_offer.for
+        )
```
Comment (gregnazario) [inline aptos-move/framework/aptos-framework/sources/account.move:596]: This seems unnecessary

File: aptos-move/framework/aptos-framework/sources/randomness.spec.move [hunk 20]
```diff
@@ -3,7 +3,9 @@ spec aptos_framework::randomness {
     spec module {
         use aptos_framework::chain_status;
         pragma verify = true;
-        invariant [suspendable] chain_status::is_operating() ==> exists<PerBlockRandomness>(@aptos_framework);
+        invariant[suspendable] chain_status::is_operating() ==> exists<PerBlockRandomness>(
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/randomness.spec.move:6]: There should be a space between `invariant` and `[suspendable]`

File: aptos-move/framework/aptos-framework/sources/account.move [hunk 21]
```diff
@@ -678,14 +813,20 @@ module aptos_framework::account {
 
         // Set `OriginatingAddress[new_auth_key] = originating_address`.
         let new_auth_key = from_bcs::to_address(new_auth_key_vector);
-        table::add(address_map, new_auth_key, originating_addr);
+        table::add(
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/account.move:816]: Why do we need this line break here?

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move (hunk 6 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:37]: Here, the line break is due to the length of the parameters inside the parentheses being too long.

File: aptos-move/framework/aptos-framework/sources/account.spec.move (hunk 8 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.spec.move:736]: We will fix this.

File: aptos-move/framework/move-stdlib/tests/vector_tests.move (hunk 10 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/tests/vector_tests.move:107]: we will fix it.

File: aptos-move/framework/move-stdlib/sources/fixed_point32.move (hunk 11 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/sources/fixed_point32.move:303]: we will fix it

File: aptos-move/framework/aptos-framework/sources/delegation_pool.move (hunk 15 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/delegation_pool.move:10]: we already fixed it https://github.com/movebit/movefmt/commit/28a55ff52096655e835e6ad748b0ee23a328716f

File: aptos-move/framework/aptos-framework/sources/randomness.spec.move (hunk 20 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/randomness.spec.move:6]: we will fix it

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move (hunk 16 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:27]: we will fix it

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 17 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:250]: we will fix it

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 21 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:816]: Here, the line break is due to the length of the parameters inside the parentheses being too long. But this rule should just use for **Function Header**, not the function call. We will fix it

File: aptos-move/framework/aptos-framework/sources/aptos_governance.move [hunk 22]
```diff
@@ -783,17 +905,36 @@ module aptos_framework::aptos_governance {
...
+        );
 
         let execution_hash = vector::empty<u8>();
         vector::push_back(&mut execution_hash, 1);
 
         create_proposal_for_test(&proposer, multi_step);
 
-        vote(&yes_voter, signer::address_of(&yes_voter), 0, true);
-        vote(&no_voter, signer::address_of(&no_voter), 0, false);
+        vote(
```
[the remark this answers is not in the document]
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aptos_governance.move:920]: Here, the line break is due to the length of the parameters inside the parentheses being too long. But this rule should just use for Function Header, not for the function call. We will fix it

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator_factory.move [hunk 23]
```diff
@@ -26,9 +26,7 @@ module aptos_framework::aggregator_factory {
     /// Creates a new factory for aggregators. Can only be called during genesis.
     public(friend) fun initialize_aggregator_factory(aptos_framework: &signer) {
         system_addresses::assert_aptos_framework(aptos_framework);
-        let aggregator_factory = AggregatorFactory {
-            phantom_table: table::new()
-        };
+        let aggregator_factory = AggregatorFactory {phantom_table: table::new()};
```
Comment (rahxephon89) [inline aptos-move/framework/aptos-framework/sources/aggregator/aggregator_factory.move:29]: This reformatting seems not necessary

File: aptos-move/framework/aptos-framework/sources/randomness.spec.move (hunk 20 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/randomness.spec.move:6]: fixed at https://github.com/movebit/movefmt/commit/d94da4a5e323435fec82a90b4ae06c6d5ed5a0a2

File: aptos-move/framework/move-stdlib/tests/vector_tests.move (hunk 10 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/tests/vector_tests.move:107]: fixed at https://github.com/movebit/movefmt/commit/d94da4a5e323435fec82a90b4ae06c6d5ed5a0a2

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 17 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:250]: fixed at https://github.com/movebit/movefmt/commit/a502614c0de751126eae46f2dcf4c90d37a8faee

File: aptos-move/framework/aptos-framework/sources/delegation_pool.move (hunk 15 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/delegation_pool.move:10]: https://github.com/movebit/movefmt/commit/28a55ff52096655e835e6ad748b0ee23a328716f

File: aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move (hunk 14 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-stdlib/sources/table_with_length.spec.move:7]: we fixed at: https://github.com/movebit/movefmt/commit/2bf7f82ca12800f4af8796fe8d3c53f303a5695e

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 21 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:816]: fixed at https://github.com/movebit/movefmt/commit/a3e29fd6f6b715962de1b65192f48d84d70641cf

File: aptos-move/framework/aptos-framework/sources/aptos_governance.move (hunk 22 above)
[the remark this answers is not in the document]
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aptos_governance.move:920]: fixed at https://github.com/movebit/movefmt/commit/a3e29fd6f6b715962de1b65192f48d84d70641cf

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 2 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:216]: fixed at https://github.com/movebit/movefmt/commit/06f2daf949649e1b4f9f6b1fb1df64ab7620a26e

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 3 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:391]: fixed at https://github.com/movebit/movefmt/commit/64839f36df83bd31864e1cb70ac38e17ef645303

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 4 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:903]: fixed at https://github.com/movebit/movefmt/commit/06f2daf949649e1b4f9f6b1fb1df64ab7620a26e

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 5 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:300]: fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move (hunk 6 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:37]: fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c

File: aptos-move/framework/aptos-framework/sources/aptos_coin.move (hunk 7 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aptos_coin.move:200]: fixed at https://github.com/movebit/movefmt/commit/9ca658736a254a5696676f95c8da7b498a90281c

File: aptos-move/framework/aptos-framework/sources/aptos_account.spec.move (hunk 9 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:155]: fixed

File: aptos-move/framework/move-stdlib/sources/fixed_point32.move (hunk 11 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/sources/fixed_point32.move:303]: fixed at https://github.com/movebit/movefmt/commit/a15b8f3b2b461a42e5f0911034bd697ef0d62439

File: aptos-move/framework/move-stdlib/sources/configs/features.spec.move (hunk 12 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/sources/configs/features.spec.move:55]: fixed

File: aptos-move/framework/move-stdlib/sources/bit_vector.move (hunk 13 above)
Comment (robinlzw) [reply aptos-move/framework/move-stdlib/sources/bit_vector.move:192]: fixed

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move (hunk 16 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:27]: fixed at https://github.com/movebit/movefmt/commit/88e7c98963e3cc88525cf31483f0ba075af4d9f7

Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move:27]: fixed at https://github.com/movebit/movefmt/commit/88e7c98963e3cc88525cf31483f0ba075af4d9f7

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 18 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:379]: fixed

File: aptos-move/framework/aptos-framework/sources/account.move (hunk 19 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/account.move:596]: fixed

File: aptos-move/framework/aptos-framework/sources/aggregator/aggregator_factory.move (hunk 23 above)
Comment (robinlzw) [reply aptos-move/framework/aptos-framework/sources/aggregator/aggregator_factory.move:29]: fixed

**Pull request closed without merging**